CHANGES.rst

```diff
@@ -8,6 +8,10 @@
 Changes
 =======
 
+Version 5.2.0 (released 2024-04-02)
```

Comment:
should be patch version no?

Reply:
Change of behaviour, I would go with minor...